spec/controllers/social_networking/nudges_controller_spec.rb

```diff
@@ -56,6 +56,25 @@ module SocialNetworking
             allow(controller).to receive(:send_sms) { nil }
           end
 
+          it "sends an email alert with the localized app name" do
+            post :create, recipientId: 123
+            allow(controller).to receive(:t).and_return("SunnySide")
+            message_subject = controller
```

Comment:
It's generally bad practice to call private methods, because you should be able to refactor/rename them without functionality or specs breaking.

As an alternative, I'd add an expectation that `NudgeMailer.nudge_email_alert` gets called with the appropriate arguments.